backend/services/queues/monthly_dump_queue_service.py

```
@@ -0,0 +1,259 @@
+        msg_str = message.get("message", "{}")
+        msg_data = json.loads(msg_str) if isinstance(msg_str, str) else msg_str
+
+        monthly_dump_id = msg_data.get("monthly_dump_id")
+        user_id = msg_data.get("user_id")
+        month = msg_data.get("month")
+        timezone_name = msg_data.get("timezone") or "UTC"
+        seed = msg_data.get("random_seed")
+        failure_count = msg_data.get("failure_count", 0)
+
+        if not monthly_dump_id or not user_id or not month:
+            logger.warning(
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)